include/inexor/vulkan-renderer/wrapper/device.hpp

```diff
@@ -7,80 +7,98 @@
 #include <vulkan/vulkan_core.h>
 
 #include <array>
-#include <cassert>
 #include <functional>
 #include <optional>
+#include <span>
 #include <string>
 
 namespace inexor::vulkan_renderer::wrapper {
 
-/// @brief A RAII wrapper class for VkDevice, VkPhysicalDevice and VkQueues.
+/// A wrapper struct for physical device data
```

Comment:
Can the struct be moved to `device.cpp` as it is not used somewhere else if iam not mistaken?